packages/twenty-front/src/modules/command-menu/pages/workflow/action/components/CommandMenuWorkflowSelectActionContent.tsx

```diff
@@ -116,6 +120,14 @@ export const CommandMenuWorkflowSelectActionContent = ({
         {t`Human Input`}
       </RightDrawerWorkflowSelectStepTitle>
       {WorkflowActionMenuItems(HUMAN_INPUT_ACTIONS, theme, handleCreateStep)}
```

Comment:
It's really strange to call `WorkflowActionMenuItems` as a function like this 😮. Instead, we should render it as a standard React component. This is not something to fix in this PR, as it existed before it; however, we definitely need to clean it up.